### DISCUSSION TITLE
Copilot chat on VSCode can not show the picture

### DISCUSSION THREAD

**Discussion by zhuchangyi** [community/community#55599] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Copilot chat on VSCode can not show the picture on Dialog box
![image](https://github.com/community/community/assets/51321351/3c0db4fe-11ea-4b7a-8f70-e446dc855d36)
It seems it's unloaded

**Comment by Akash1134** [comment, marked as answer]
Hi @zhuchangyi ,

Are you approved for the Copilot Chat waitlist and have got a notification for the same?

If not so, please note that feature of Copilot X(Including Copilot Chat)  has not yet been released and we do not have a timeline for availability. As soon as more information is available, we look forward to sharing.

In the meantime, if you are currently a Copilot subscriber, OSS, or Student/Faculty user -  you can [sign up for the Copilot X Technical Preview](https://github.com/github-copilot/chat_waitlist_signup) waitlist.

Thanks!

**Reply by zhuchangyi** [reply]
Hi @Akash1134 
I have been approved for the Copilot Chat waitlist already. I tried this later this afternoon and the picture could be loaded.
I suppose this could be the network probelm. 
By the way, the Copilot Chat is really amazing and it helps me a lot .Thank you  for your response and the provided information! I will closely follow any further updates or developments and be ready to participate at any time.

**Reply by Akash1134** [reply]
Copilot Chat is indeed an amazing addition. Glad to know it was only a network problem. 😃 Thanks!
